### PR TITLE
Build exe module

### PR DESCRIPTION
**Pull request by wenaro**

### CODE REVIEW

**Review by NdV66 — CHANGES_REQUESTED** [review]
fix travis

**Comment by wenaro** [comment]
"/home/travis/.m2/repository/com/akathist/maven/plugins/launch4j/launch4j-maven-plugin/1.5.2/launch4j-maven-plugin-1.5.2-workdir-linux/bin/windres": error=2, No such file or directory
i think that we need to add this file for proper build exe

**Comment by wenaro** [comment]
Build is successful when I added file below
[windres.zip](https://github.com/inzOprogramowaniaUZ/violetumleditor/files/788310/windres.zip)

**Comment by wenaro** [comment]
But I dont think that is good way for fix it. What do you think?

**Comment by wenaro** [comment]
https://github.com/travis-ci/travis-ci/issues/5176

> I ran into the same problem. This is what you need to add to your .travis.yml to install the missing 32-> bit libs:
> sudo: required
> before_script:
>   - sudo apt-get update
>   - sudo apt-get install lib32z1 lib32ncurses5

**Comment by wenaro** [comment]
https://github.com/lukaszlenart/launch4j-maven-plugin

Q: Can I use Launch4j on 64bit OS?

A: Yes but you will have to install these libs to avoid problems:

lib32z1
lib32ncurses5
lib32bz2-1.0 ( (has been ia32-libs in older Ubuntu versions)
zlib.i686
ncurses-libs.i686
bzip2-libs.i686

**Comment by wenaro** [comment]
I think now we have enough knowledge to decided whether we want to build this module. So I look forward to your opinions.

**Comment by RoughTomato** [comment]
Wait, is this the problem with travis or with build in general?
If application require those libraries we should just throw an error message informing user that he will need to install these or their equivalent for his distribution. I don't think we want to support downloading those libraries. Considering how many package managers there are in linux & unix world.

I believe that adding that information will be sufficient.

Maybe making this a custom build might be a better idea?
e.g `mvn clean install-exe` or something (I'm not sure how this works with maven, but it's quite easy with gradle). If this is too difficult (i.e. might complicate build process too much to be comprehensive) or time consuming (i.e. we won't meet deadline) let's just skip this one.

P.S. I'll throw additional information about this requirement in documentation in build section. But this needs to be resolved as well.

File: pom.xml [hunk 1]
```diff
@@ -22,8 +23,8 @@
     <module>violetproduct-swing</module>
     <module>violetproduct-deb</module>
 
-    <!-- Modules 'exe' and 'web' are commented out because they're causing problems during project build-->
-    <!--<module>violetproduct-exe</module>-->
+    <!-- Module 'web' is commented out because he's causing problems during project build-->
```
Comment (RoughTomato) [inline pom.xml]: "it's" not "he's". Module is impersonal.

**Review by RoughTomato — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (wenaro) [reply pom.xml]: yes, you are right.

**Comment by wenaro** [comment]
I don't know how to make custom build (if it is possible on maven), so I think the best way is just skip this one feature, and describe to Ph.D Hladowski problems with this module.

**Comment by abizukojc** [comment]
I agreee with @wenaro

**Pull request closed without merging**